pkg/scheduler/internal/cache/cache.go

```diff
@@ -183,9 +183,9 @@ func (cache *schedulerCache) Dump() *Dump {
 		nodes[k] = v.info.Clone()
 	}
 
-	assumedPods := make(map[string]bool, len(cache.assumedPods))
-	for k, v := range cache.assumedPods {
-		assumedPods[k] = v
+	assumedPods := make(sets.String, len(cache.assumedPods))
+	for k := range cache.assumedPods {
+		assumedPods.Insert(k)
```

Comment:
We're cloning `cache.assumedPods` here, so suggest to return the cloned set directly in L193:

```go
return &Dump{
    Nodes:       nodes,
    AssumedPods: cache.assumedPods.Union(nil),
}
```